yajsapi/executor/index.ts

```diff
@@ -17,7 +17,9 @@ import * as rest from "../rest";
 import { Agreement, OfferProposal, Subscription } from "../rest/market";
 import { AgreementsPool } from "./agreements_pool";
 import { Allocation, DebitNote, Invoice } from "../rest/payment";
-import { ActivityService, CommandExecutionError } from "../rest/activity";
+import { CommandExecutionError } from "../rest/activity";
+
+import { ActivityFactory, Activity } from "../mid-level-api/activity";
```

Comment:
I'd still prefer not to have the "mid-level-api" named as a specific module -> it's, to an extent, artificial and non-functional ... I'd just put this `activity` module in the same directory tree as other specialized modules, that is, as: `yajsapi/activity`... 

I'm not pushing for this to happen as part of this pull request though, just wouldn't like for this to remain in place after we finalize the whole "activity" saga